set-mongo/src/main/java/eu/europeana/set/mongo/service/PersistentUserSetServiceImpl.java

```diff
@@ -222,6 +222,60 @@ private List<DBObject> getDistinctCountPipeline(String groupField, String type)
 
 		return distinctCountPipeline;
 	}
+	
+    /**
+     * Creates an aggregation pipeline for 2 metrics on the Entity Galleries (Sets):
+     * NumberOfEntitySets and NumberOfItemsInEntitySets.     * 
+     * query :
+     * [ {  $match: { type: "EntityBestItemsSet" }},
+     *   {  $group: { 
+     *          _id: null,
+     *          totalEntityGalleries: { $sum: 1 },
+     *          totalItemsInEntityGalleries: { $sum: { $size:"$items" } }           
+     *      }
+     *   }]
+     * @return
+     */
+	@Override
+    public long[] getEntityGalleriesMetrics() {
```

Comment:
this method is not needed, please reuse the getDistinctCountPipeline() method with proper input parameters, in order to avoid duplicated code